apps/website/src/app/api/virtual-tickets/[event]/[user]/og/route.tsx

```diff
@@ -0,0 +1,109 @@
+import { ImageResponse } from "@vercel/og";
```

Comment:
Ideally the two handlers should be DRYed, and abstracted to be usable for future events.